src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1297,29 +1275,22 @@ By default, we include the following package managers in each runtime.
 
 # Using Appwrite in a function {% #using-appwrite %} 
 Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies. 
-Authenticating with Appwrite is done via an API key or a JWT token. 
-API keys must be generated and exported as an [environment variable](/docs/advanced/self-hosting/environment-variables).
+Authenticating with Appwrite is done via a dynamic API key or a JWT token. 
 
 You can read more about authentication in the [JWT login](/docs/products/auth/jwt) section of the docs.
 
 ## Using with API key {% #using-api-key %} 
-API keys have defined scopes when you create them. 
-They ignore permissions and operate without a sessions. 
-Use API keys if the function should act as an admin type role, instead of acting on behalf of a user. 
-Pass in your API key as an environment variable. 
-Never share API keys with users.
+Appwrite functions use dynamic API keys, allowing functions to ignore permissions and operate without a session.
+You can only use dynamic API keys inside Appwrite functions.
+Inside Appwrite functions, dynamic or regular API keys don't have to be created or managed.
+Instead, use the function settings to set what it is allowed to do.
 
 {% multicode %}
 ```server-nodejs
 import { Client, Databases, ID } from 'node-appwrite';
 
 export default async ({ req, res, log, error }) => {
-
-    const client = new Client()
-        .setEndpoint('https://cloud.appwrite.io/v1')
-        .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID)
-        .setKey(process.env.APPWRITE_API_KEY);
-
+    const client = new Client().setKey(req.headers['x-appwrite-key']); // Set API key; it's automatically provided by Appwrite
```

Comment:
Let's keep the multiline like before so we can make it clear the comment is for the `setKey()` part.